### ISSUE TITLE
/usr/bin/ld: cannot find -lpurple

### ISSUE THREAD

**Issue by AgentJeffy** [dequis/purple-facebook#376]
Wow this has been difficult. I'm trying to get this set up on Puppy Linux 6.0.6

When I run the make command I get the following:

``root# make install
Making install in pidgin/libpurple/protocols/facebook
make[1]: Entering directory `/root/purple-facebook-0.9.5-9ff9acf9fa14/pidgin/libpurple/protocols/facebook'
  CCLD     libfacebook.la
/usr/bin/ld: cannot find -lpurple
collect2: error: ld returned 1 exit status
make[1]: *** [libfacebook.la] Error 1
make[1]: Leaving directory `/root/purple-facebook-0.9.5-9ff9acf9fa14/pidgin/libpurple/protocols/facebook'
make: *** [install-recursive] Error 1
root# `
`

Needless to say I've exhausted all other thought processes before coming here for some assistance.

**Comment by AgentJeffy** [comment]
Okay, updated the code to be more legible. Again, apologize for the original poor formatting it was in. I'm genuinely hoping that there is something simple I'm overlooking here. But I am back at it again. By the time I'm done here I should be able to tell other Puppy users how to get set up with Purple Facebook.

**Comment by EionRobb** [comment]
Do you have libpurple installed?

**Comment by AgentJeffy** [comment]
Yes, I currently have these installed:

libpurple-bin_2.10.9
libpurple-dev_2.10.9
libpurple0_2.10.9

**Comment by EionRobb** [comment]
And is libpurple.so in location from `pkg-config --libs purple-2` ?

**Comment by AgentJeffy** [comment]
I checked, and at /usr/lib/purple-2 there are a bunch of files pertaining to it, but no file named libpurple.so @EionRobb

**Comment by AgentJeffy** [comment]
And in /usr/lib/pkgconfig there is no libpurple file.

**Comment by EionRobb** [comment]
No, what is the output from running `pkg-config --libs purple-2` in a terminal

**Comment by AgentJeffy** [comment]
```
root# pkg-config --libs purple-2
Package purple-2 was not found in the pkg-config search path.
Perhaps you should add the directory containing `purple-2.pc'
to the PKG_CONFIG_PATH environment variable
No package 'purple-2' found
root#
```

@EionRobb But I know it's actually installed... That was the /usr/lib/purple-2 folder I was checking earlier. Perhaps I need to instruct it where to find the files?

**Comment by AgentJeffy** [comment]
Or is there no way to do that?

**Comment by EionRobb** [comment]
Or find which package installs purple-2.pc and install that?

**Comment by AgentJeffy** [comment]
Isn't that supposed to come with the purple dev pack?

**Comment by AgentJeffy** [comment]
So when I run the code `pkg-config --libs purple` I get a return message of `-lpurple -lglib-2.0` so I'm really at a loss here. I have no idea what's going on.

**Comment by AgentJeffy** [comment]
Here is the complete code when `make` is run after a successful `./configure`

```
root# make
Making all in pidgin/libpurple/protocols/facebook
make[1]: Entering directory `/root/purple-facebook-0.9.5-9ff9acf9fa14/pidgin/libpurple/protocols/facebook'
  CC       libfacebook_la-marshal.lo
  CC       libfacebook_la-api.lo
  CC       libfacebook_la-data.lo
  CC       libfacebook_la-facebook.lo
  CC       libfacebook_la-http.lo
  CC       libfacebook_la-json.lo
  CC       libfacebook_la-mqtt.lo
  CC       libfacebook_la-thrift.lo
  CC       libfacebook_la-util.lo
  CC       ../../libfacebook_la-http.lo
  CC       ../../libfacebook_la-purple-socket.lo
  CCLD     libfacebook.la
/usr/bin/ld: cannot find -lpurple
collect2: error: ld returned 1 exit status
make[1]: *** [libfacebook.la] Error 1
make[1]: Leaving directory `/root/purple-facebook-0.9.5-9ff9acf9fa14/pidgin/libpurple/protocols/facebook'
make: *** [all-recursive] Error 1
```
